odpf/proton/metadata/resource/OptimusLog.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package odpf.proton.metadata.resource;
+
+option java_package = "io.odpf.proton.metadata.resource";
+option java_outer_classname = "OptimusLog";
+option go_package = "github.com/odpf/proton/metadata/resource";
+
+import "google/protobuf/timestamp.proto";
+
+message OptimusJobMetadataKey {
+  string urn = 1;
+}
+
+message OptimusJobMetadata {
+  string urn = 1;
```

Comment:
How exactly do you define `urn` here? Is this just the name? Or it's really unique across projects?

Reply:
as discussed we'll keep it like this, and it'll be unique across all tenants.
```
tenant_name::job/<name>
tenant_name::table/<name>
tenant_name::view/<name>
```